pkg/test/framework/components/echo/kube/templates/echo_deployment.template

```diff
@@ -0,0 +1,213 @@
+{{- $revVerMap := .Revisions }}
```

Comment:
nit: name the file with .yaml suffix instead of .template, most IDEs recognize helm templates suffixed with .yaml

Reply:
+1. I've also seen templates use the `.yaml.tmpl` extension, but `yaml` will be more easily understood by IDEs by default.

Reply:
Also, let's just call this file `deployment.yaml`. The `echo` part is already understood.